Event_Example.py

```diff
@@ -0,0 +1,72 @@
+import requests
+import json
+import os
+import sys
+
+# API endpoint and headers
+url = "https://api.ailicia.live/v1/events"

```

Comment:
We recently changed the url of the API to https://api.getailicia.com instead of https://api.ailicia.live, if you can make the update